web/src/tippyjs.js

```diff
@@ -105,6 +105,19 @@ export function initialize() {
         target: ".tippy-zulip-tooltip",
     });
 
+    delegate("body", {
+        target: ".tippy-username-tooltip",
+        placement: "top",
+        onShow(instance) {
+            instance.setContent(instance.reference.textContent);
+            instance.popper.querySelector(".tippy-content").classList.add("tooltip_username");
```

Comment:
We can create a handlebar which takes `class_name` as context and use it since this can be slow to take effect which can cause user to see a transition state where the css is not applied to being applied.

Reply:
Are you suggesting something like this?
```
const context = {
    username: instance.reference.textContent,
    class_name:'tooltip_username'
};

instance.setContent(parse_html(
  $t_html(
    {defaultMessage: "<span>{username}</span>"},
    context,
  )
));
```

The issue with implementing this approach is that including `class='{class_name}'` within the `span` tag results in a parsing error. Is there a viable solution available to resolve this problem?

Reply:
Are you suggesting something like this?
```
const context = {
    username: instance.reference.textContent,
    class_name:'tooltip_username'
};

instance.setContent(parse_html(
  $t_html(
    {defaultMessage: "<span>{username}</span>"},
    context,
  )
));
```

The issue with implementing this approach is that including `class='{class_name}'` within the `span` tag results in a parsing error. Is there a viable solution available to resolve this problem?

Reply:
I meant we can create a template like `message_inline_image_tooltip`